package/Dockerfile

```diff
@@ -1,39 +1,66 @@
 # Final micro image
 FROM registry.suse.com/bci/bci-micro:15.5 AS micro
 
+# Image that provides cross compilation tooling.
+FROM --platform=$BUILDPLATFORM tonistiigi/xx:latest as xx
 
 # Temporary build stage
-FROM registry.suse.com/bci/bci-base:15.5 AS builder
+FROM registry.suse.com/bci/golang:1.21 AS builder
 
 # Define build arguments
-ARG kube_bench_version=0.7.0
-ARG sonobuoy_version=0.57.0
+ARG KUBE_BENCH_VERSION
+ARG SONOBUOY_VERSION
+ARG KUBECTL_VERSION
 
-ARG kubectl_version=1.28.0
-ARG ARCH
+ARG TARGETARCH
 
 # Install system packages using builder image that has zypper 
 COPY --from=micro / /chroot/
 
 # Install kubectl into micro
-RUN curl -Lo /chroot/usr/local/bin/kubectl "https://dl.k8s.io/release/v${kubectl_version}/bin/linux/${ARCH}/kubectl" && chmod +x /chroot/usr/local/bin/kubectl
+ADD --chown=root:root --chmod=0755 \
+    "https://dl.k8s.io/release/v${KUBECTL_VERSION}/bin/linux/${TARGETARCH}/kubectl" \
+    /chroot/usr/local/bin/kubectl
 
 ## Install Sonobuoy into micro
-RUN curl -sLf "https://github.com/vmware-tanzu/sonobuoy/releases/download/v${sonobuoy_version}/sonobuoy_${sonobuoy_version}_linux_${ARCH}.tar.gz" | tar -xvzf - -C /chroot/usr/bin sonobuoy
+RUN curl -sLf "https://github.com/vmware-tanzu/sonobuoy/releases/download/v${SONOBUOY_VERSION}/sonobuoy_${SONOBUOY_VERSION}_linux_${TARGETARCH}.tar.gz" | tar -xvzf - -C /chroot/usr/bin sonobuoy
 
 ## Install kube-bench into micro
-RUN curl -sLf "https://github.com/aquasecurity/kube-bench/releases/download/v${kube_bench_version}/kube-bench_${kube_bench_version}_linux_${ARCH}.tar.gz" | tar -xvzf - -C /chroot/usr/bin
+RUN curl -sLf "https://github.com/aquasecurity/kube-bench/releases/download/v${KUBE_BENCH_VERSION}/kube-bench_${KUBE_BENCH_VERSION}_linux_${TARGETARCH}.tar.gz" | tar -xvzf - -C /chroot/usr/bin
 
 ## Copy the files within /cfg straight from the immutable GitHub source to /etc/kube-bench/cfg/ into micro
 RUN mkdir -p /chroot/etc/kube-bench/ && \
-    curl -sLf "https://github.com/aquasecurity/kube-bench/archive/refs/tags/v${kube_bench_version}.tar.gz" | \
-    tar xvz -C /chroot/etc/kube-bench/ --strip-components=1 "kube-bench-${kube_bench_version}/cfg"
+    curl -sLf "https://github.com/aquasecurity/kube-bench/archive/refs/tags/v${KUBE_BENCH_VERSION}.tar.gz" | \
+    tar xvz -C /chroot/etc/kube-bench/ --strip-components=1 "kube-bench-${KUBE_BENCH_VERSION}/cfg"
```

Comment:
Great idea! Made changes to that effect. Points of note:
- Changed all dependencies and versions to their own file: `hack/make/deps.mk`, as `Makefile` started to look too cluttered. This will also help with not breaking renovate for the time being.
- Renovate supports this for github-releases. For things like `kubectl` we may need to find a better way around it (maybe `updatecli` or maybe a [custom datasource](https://docs.renovatebot.com/modules/datasource/custom/)).

Reply:
A new issue on renovate-config has been created to track the [checksums](https://github.com/rancher/renovate-config/issues/206) changes required.